src/components/Properties/PropertyActions.vue

```diff
@@ -28,9 +28,10 @@
 			</template>
 			{{ t('contacts', 'Delete') }}
 		</ActionButton>
-		<Actions :is="action"
-			v-for="(action, index) in actions"
-			:key="index" />
+		<component v-for="action in actions"
+			:is="action"
+			:key="action.name"
+			:component="propertyComponent" />
```

Comment:
This prop is already passed by the parent. It was just missing a definition here.